app/components/modal-clipboard/modal-clipboard.tsx

```diff
@@ -72,12 +76,13 @@ export const ModalClipboard = () => {
   const navigation = useNavigation()
 
   const open = async () => {
-    dismiss()
+    modalClipboardVisibleVar(false)
     navigation.navigate("sendBitcoin", { payment: await Clipboard.getString() })
   }
 
-  const dismiss = () => {
+  const dismiss = async () => {
     modalClipboardVisibleVar(false)
+    saveString(LAST_CLIPBOARD_PAYMENT, await Clipboard.getString())
```

Comment:
Why are we not using Apollo for storing this?

Reply:
After thinking about this more I believe continuing to use AsyncStorage directly with saveString and loadString does not seem bad for client only state. In order to persist the cache we are using [apollo3-cache-persist](https://github.com/apollographql/apollo-cache-persist) and [Async Storage](https://github.com/react-native-async-storage/async-storage). Thoughts?

Reply:
The alternative is trying to get something like this to work: https://www.apollographql.com/docs/react/local-state/managing-state-with-field-policies/#storing-local-state-in-the-cache